config/environments/development.rb

```diff
@@ -37,11 +37,6 @@
 
   config.action_controller.forgery_protection_origin_check = ENV['DISABLE_FORGERY_REQUEST_PROTECTION'].nil?
 
-  ActiveSupport::Logger.new($stdout).tap do |logger|
-    logger.formatter = config.log_formatter
-    config.logger = ActiveSupport::TaggedLogging.new(logger)
```

Comment:
This style config remains in place for production, but the default is basically this already for dev mode.